client/src/screens/events/Detail.js

```diff
@@ -144,17 +240,16 @@ class Detail extends Component {
         id: props.event.id,
         locationLatitude: latitude,
         locationLongitude: longitude,
+        locationTime: position.timestamp,
       })
       .then(() => {
-        console.log('location updated');
+        console.log('location change sent');
       })
       .catch((err) => {
-        console.log(`Failed to update location: ${latitude},${longitude} (${err})`);
+        console.log(`Failed to send location change: ${latitude},${longitude} (${err})`);
       });
   }, 4000);
 
-  geoWatch = null;
```

Comment:
why'd you remove this? I believe you should initialise it somewhere if it's going to be used. Checking for undefined should be avoided imo.

Reply:
good point

Reply:
?

Reply:
its back in.